### DISCUSSION TITLE
Leave `label` in sidebar empty or absent and get the value from header of document

### DISCUSSION THREAD

**Discussion by CanadianBeaver** [facebook/docusaurus#10134] [General]
Is it possible to leave `label` empty and get the actual value from header in .markdown? I mean like for `items`  array does it:

```
mySidebar: [
    {
      type: 'category',
      label: 'My title', // <- here, value is not getting from header of document 
      link: {
        type: 'doc',
        id: 'url/README',
      },
      items: [
        'url/Doc1', // <- these values become from header, that is good
        'url/DocN'
      ]
    }
  ]
```

**Comment by slorber** [comment]
We have a `sidebar_label` front matter, but it only works for docs in auto-generated sidebars afaik.

I don't think this is possible atm, and according to what I know of the code, it looks like adding support for this would be challenging and require some refactoring.
